netlify/functions/feedback-management.js

```diff
@@ -137,9 +201,60 @@ exports.handler = async (event, context, callback) => {
         }`,
       ].join('\n\n');
 
-      await slackClient.chat.postMessage({
+      const githubNewIssueParams = new URLSearchParams({
+        title: '[ ] Docs feedback',
+        body: `Feedback received:
+${comment}
+
+from ${commmentSectionURL}
+`,
+      });
+
+      await app.client.chat.postMessage({
         channel: getSlackChannelId(currentLocationURL, { isDesignFeedback }),
-        text: simpleSlackMessage,
+        text: simpleSlackMessage, // Fallback for notification
+        blocks: [
+          {
+            type: 'section',
+            text: {
+              type: 'mrkdwn',
+              text: simpleSlackMessage,
+            },
+          },
+          {
+            type: 'actions',
+            elements: [
+              {
+                type: 'button',
+                text: {
+                  type: 'plain_text',
+                  text: 'Create issue',
+                  emoji: true,
+                },
+                url: `https://github.com/mui/${getGitHubRepo(currentLocationURL, {
```

Comment:
Is there a way from the frontend to pass `process.env.SOURCE_CODE_REPO` to the Netlify backend function? It sounds more future-proof than reconstruction URLs.